packages/boxed-expression-component/src/expressions/ExpressionDefinitionRoot/ExpressionContainer.tsx

```diff
@@ -44,11 +43,7 @@ export const ExpressionContainer: React.FunctionComponent<ExpressionContainerPro
   const { setExpression } = useBoxedExpressionEditorDispatch();
 
   const getDefaultExpressionDefinition = useCallback(
-    (isNested, logicType, prev, containerWidth) => {
-      return isNested
-        ? getDefaultExpressionDefinitionByLogicType(logicType, prev, containerWidth)
-        : beeGwtService!.getDefaultExpressionDefinition(logicType);
-    },
+    (logicType) => beeGwtService!.getDefaultExpressionDefinition(logicType),
     [beeGwtService]
   );
```

Comment:
Maybe this can be inlined? Don't need to wrap it in a `useCallback` anymore..

Reply:
Done!